GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -876,20 +932,19 @@ internal override void ComputeNPCCombatReplayActors(NPC target, ParsedEvtcLog lo
                         foreach (EffectEvent effect in kralkBeamEffects)
                         {
                             int indicatorDuration = 2000;
-                            int aoeDuration = 5000;
-                            int indicatorStart = (int)effect.Time;
-                            int aoeStart = indicatorStart + indicatorDuration;
-                            int aoeEnd = Math.Min(aoeStart + aoeDuration, (int)target.LastAware);
-                            replay.AddDecorationWithGrowing(new RectangleDecoration(700, 2900, (indicatorStart, aoeEnd), Colors.Orange, 0.2, new PositionConnector(effect.Position)), aoeStart);
+                            long growingEnd = effect.Time + indicatorDuration;
```

Comment:
The lifespan of the rectangle has been reduced, intended?

Reply:
Yeah I've changed it to appear only as indicator just like in game, having the rectangle shown as dangerous in spots where there's no damage isn't ideal imo